### DISCUSSION TITLE
Configurare accesso remoto SSL con Let's Encrypt senza l'utilizzo di DuckDNS

### DISCUSSION THREAD

**Discussion by morfeus1986** [indomus/forum#227] [GENERALE - Discussioni generiche]
Salve,

Vorrei configurare l'accesso remoto in https ad HA. Non voglio usare DuckDNS sia perchè spesso è down e sia perchè ho un fritz e posso usare il loro servizio xxxx.myfritz.com per l'associazione IP/Nome.  Non ho problemi a configurare i port forwarding sul router, ma non riesco a trovare una guida per configurare l'add-on di Let's Encrypt. Qualcuno può aiutarmi ?

Nello specifico: 

![image](https://user-images.githubusercontent.com/79855524/116269599-a754c300-a77e-11eb-9956-e97ec01a8149.png)

**Comment by 50494554524F** [comment]
la guida degli add-on è nel tab "documentazione" che trovi proprio nella pagina di configurazione...
